babel/messages/extract.py

```diff
@@ -544,7 +545,11 @@ def extract_javascript(fileobj, keywords, comment_tags, options):
     :param options: a dictionary of additional options (optional)
                     Supported options are:
                     * `jsx` -- set to false to disable JSX/E4X support.
-                    * `template_string` -- set to false to disable ES6 template string support.
+                    * `template_string` -- if `True`, supports gettext(`key`)
+                    * `parse_template_string` -- if `True` will parse the
+                                                 contents of javascript
+                                                 template strings.
+    :param lineno: the line number to start from (optional)
```

Comment:
This should probably be clarified somehow to note that it doesn't change how parsing works, but just offsets the `lineno` in the yielded items?
```suggestion
    :param lineno: line number offset (for parsing embedded fragments)
```

Also, maybe it should be in `options` all in all, so as not to change the extractor function signature?